proto/api_v3/query_service.proto

```diff
@@ -40,47 +40,64 @@ message GetTraceRequest {
   bool raw_traces = 4;
 }
 
-// Query parameters to find traces. Except for num_traces, all fields should be treated
-// as forming a conjunction, e.g., "service_name='X' AND operation_name='Y' AND ...".
+// Query parameters to find traces. All fields should be treated
+// as forming a conjunction, e.g., "service_name='X' AND operation_name='Y' AND ..."
+// with the exception of `search_depth` and `raw_traces`.
+//
 // All fields are matched against individual spans, not at the trace level.
 // The returned results contain traces where at least one span matches the conditions.
-// When num_traces results in fewer traces returned, there is no required ordering.
-//
-// Note: num_traces should restrict the number of traces returned, but not all backends
-// interpret it this way. For instance, in Cassandra this limits the number of _spans_
-// that match the conditions, and the resulting number of traces can be less.
-//
-// Note: some storage implementations do not guarantee the correct implementation of all parameters.
+// There is no guaranteed ordering of the returned results.
 //
+// Note: Some storage implementations do not guarantee the correct implementation of all parameters.
 message TraceQueryParameters {
+  // `service_name` filters traces generated by a specific service.
   string service_name = 1;
+
+  // `operation_name` filters traces generated by a specific operation.
   string operation_name = 2;
 
-  // Attributes are matched against Span and Resource attributes.
-  // At least one span in a trace must match all specified attributes.
+  // `attributes` contains key-value pairs where the key is the attribute name
+  // and the value is its string representation. Attributes are matched against
```

Comment:
> is its string representation

what happens if stored attribute is not a string, but a number?

Reply:
@yurishkuro This is a good question and I was wondering the same thing as well. It looks like in the v1 api - only string "tags" are supported. For v2 - should this instead be OTEL's attribute type? 

```proto
repeated opentelemetry.proto.common.v1.KeyValue attributes = 9
```

Reply:
Will be making that change for remote storage api v2